src/Greeting.js

```diff
@@ -0,0 +1,34 @@
+// src/Greeting.js
+
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { fetchGreeting } from './redux/actions';
+
+const Greeting = ({ greeting, fetchGreeting }) => {
+  useEffect(() => {
+    fetchGreeting();
+  }, [fetchGreeting]);
```

Comment:
- [ ] You are fetching data directly inside the useEffect hook, You are supposed to dispatch the action using `useDispatch` from `react-redux` or `redux-toolkit` libraries.

![image](https://github.com/Kingno3l/Hello-React-New-Front-End/assets/68796232/731d5a2c-adb8-4668-9076-b29a809388ee)

Hint:
This is how to properly dispatch an action with useEffect:
```javascript
import { useDispatch } from 'react-redux';
import { fetchGreeting } from './redux/actions'

const useFetching = (someFetchActionCreator) => {
  const dispatch = useDispatch();
  useEffect(() => {
    dispatch(fetchGreeting());
  }, [])
}
```